code/modules/mining/abandoned_crates.dm

```diff
@@ -1,4 +1,4 @@
-//Originally coded by ISaidNo, later modified by Kelenius. Ported from Baystation12.
+//riginally coded by ISaidNo, later modified by Kelenius. Ported from Baystation12.
```

Comment:
I dont know what the O in Originally did to deserve being deleted but you should probably fix it.